app/views/application_pets/index.html.erb

```diff
@@ -1,8 +1,9 @@
 <p> Here are all the applications for this pet </p>
 
-
-<% @applications.each do |application| %>
-    <%= link_to "#{application.name}", "/applications/#{application.id}" %>
+<% if @applications.length > 0 %>
```

Comment:
adding an if else statement to deal with the case where no applications have been submitted